features/admins/users/admin_abuse_users.feature

```diff
@@ -183,6 +183,24 @@ Feature: Admin Abuse actions
       And there should be no bookmarks on the work "Not Spam"
       And there should be no comments on the work "Not Spam"
 
+  Scenario: A permabanned spammer's comments' replies from others should stay visible
+    Given I have a work "Generic Work"
+      And a comment "I like spam" by "Spamster" on the work "Generic Work"
+      And a reply "I don't :(" by "NotSpamster" on the work "Generic Work"
+    When I am logged in as a "policy_and_abuse" admin
+      And I go to the user administration page for "Spamster"
+      And I choose "Spammer: ban and delete all creations"
+      And I press "Update"
+    Then I should see "permanently suspended"
+      And the user "Spamster" should be permanently banned
+      And I should see "I like spam"
+    When I press "Yes, Delete All Spammer Creations"
+    Then I should see "All creations by user Spamster have been deleted."
+    When I go to the work comments page for "Generic Work"
+    Then I should not see "I like spam"
+      And I should see "(Previous comment deleted.)"
+      And I should see "I don't :("
```

Comment:
If you follow my suggestion regarding `mark_as_spam!` then these lines won't work. However, the reply should still be viewable with `When I view the latest comment`. The name of the Scenario should probably be updated to something like "accessible" since it won't be visible on the work itself.